scripts/globals/job_utils/dragoon.lua

```diff
@@ -165,10 +193,10 @@ end
 xi.job_utils.dragoon.abilityCheckAngon = function(player, target, ability)
     local id = player:getEquipID(xi.slot.AMMO)
 
-    if id == 18259 then
+    if id == xi.items.ANGON then
```

Comment:
Missing the associated require. `require("scripts/globals/items")`

Reply:
Done and sorted the requires while I was there